packages/components/organisms/f-status-banner/src/components/StatusBanner.vue

```diff
@@ -1,36 +1,19 @@
 <template>
-    <div>
-        <header-component
-            data-test-id="header-component"
-            :locale="componentLocale"
-            is-transparent />
-
-        <main-banner-container
-            data-test-id="main-banner-container"
-            :base-hero-url="baseHeroUrl"
-            :copy="copy"
-            :locale="componentLocale" />
-
-        <footer-component
-            data-test-id="footer-component"
-            :locale="componentLocale" />
-    </div>
+    <main-banner-container
```

Comment:
QQ now this is just calling a banner container component. Could that component just be included here now to avoid the need for two `.vue` files, or is there a reason to route it through the `StatusBanner.vue` file and then into the `MainBannerContainer.vue`?

Reply:
Do you mean combine the `PageBanner.vue` & `MainBannerContainer.vue` components into this file and delete those? It would also mean moving the logic from the other components into here too. I wanted `MainBannerContainer.vue` to be the thing you pass props through on, so when we have other status pages requirements we can control things inside of the main container rather than keeping the logic in one file. Can refactor tho and move it all here as we don't have that requirement yet?

Reply:
I meant just combing what's in `MainBannerContainer.vue` into this component and keeping PageBanner.vue as is, but it could be useful to use this as a shell for different status layouts as you mention to keep the base component cleaner.

Maybe keep it as it is for now and once we get the requirements for other layouts we can always revisit it if needed.